lighty-modules/lighty-aaa-aggregator/lighty-aaa/src/test/java/io/lighty/aaa/ShiroInitializationTest.java

```diff
@@ -0,0 +1,37 @@
+package io.lighty.aaa;
+
+
+import io.lighty.aaa.config.AAAConfiguration;
+import io.lighty.aaa.util.AAAConfigUtils;
+import io.lighty.server.LightyServerBuilder;
+import org.mockito.Mock;
+import org.mockito.Spy;
+import org.opendaylight.aaa.api.CredentialAuth;
+import org.opendaylight.aaa.api.PasswordCredentials;
+import org.opendaylight.mdsal.binding.api.DataBroker;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+
+public class ShiroInitializationTest {
+
+    private static final AAAConfiguration AAA_CONFIGURATION = AAAConfigUtils.createDefaultAAAConfiguration();
+
+    @Mock
+    private LightyServerBuilder server;
+    @Mock
+    private DataBroker bindingDataBroker;
+    @Mock
+    private CredentialAuth<PasswordCredentials> credentialAuth;
+    @Spy
```

Comment:
We are testing his, right? No need to spy.

Reply:
redundant spy removed